### ISSUE TITLE
Release v0.4.1

### ISSUE THREAD

**Issue by klihub** [containers/nri-plugins#288]
## Release Process

- [x] Create and push new release branch
- Local release preparations (on the release branch)
  - [x] Run e2e tests
  - [x] Sync/tidy up dependencies.
    - [x] Run `go mod tidy`.
    - [x] Run `git commit -m 'go.mod,go.sum: update dependencies.' go.{mod,sum}`, if necessary.
  - [x] Run `git tag -a -m "NRI plugins $VERSION" $VERSION`.
- Publishing
  - [x] Push the tag with `git push $VERSION`.
  - [x] Check that new container images are published for the tag.
    - ```
      for i in nri-resource-policy-topology-aware nri-resource-policy-balloons nri-resource-policy-template nri-config-manager nri-memory-qos nri-memtierd; do \
          skopeo inspect --format "$i: {{.Digest}}" docker://ghcr.io/containers/nri-plugins/$i:$VERSION;
      done
      ```
  - [x] Finalize the new *draft* release created by CI
    - [x] Check that all artefacts (Helm charts) were uploaded
    - [x] Write the change log to the release.
    - [x] Get the change log OK'd by other maintainers.
  - [x] Publish the draft as a release.
  - [x] Verify that the Helm repo was updated
  - [x] Add a link to the tagged release in this issue.
- [x] Close this issue.

**Comment by klihub** [comment]
@askervin, @marquiz, @kad, @fmuyassarov [PTAL]( https://github.com/containers/nri-plugins/releases/tag/untagged-c7795a5482bc68703b77) or [edit](https://github.com/containers/nri-plugins/releases/edit/untagged-c7795a5482bc68703b77) to finalize the draft changelog.

**Comment by fmuyassarov** [comment]
> ## Release Process
> 
>     * [x]  Create and push new release branch
> 
>     * Local release preparations (on the release branch)
>       
>       * [x]  Run e2e tests
>       * [x]  Sync/tidy up dependencies.
>         
>         * [x]  Run `go mod tidy`.
>         * [x]  Run `git commit -m 'go.mod,go.sum: update dependencies.' go.{mod,sum}`, if necessary.
>       * [x]  Run `git tag -a -m "NRI plugins $VERSION" $VERSION`.
> 
>     * Publishing
>       
>       * [x]  Push the tag with `git push $VERSION`.
>       * [x]  Check that new container images are published for the tag.
>         
>         * ```
>           for i in nri-resource-policy-topology-aware nri-resource-policy-balloons nri-resource-policy-template nri-config-manager nri-memory-qos nri-memtierd; do \
>               skopeo inspect --format "$i: {{.Digest}}" docker://ghcr.io/containers/nri-plugins/$i:$VERSION;
>           done
>           ```
>       * [ ]  Finalize the new _draft_ release created by CI
>         
>         * [x]  Check that all artefacts (Helm charts) were uploaded
>         * [x]  Write the change log to the release.
>         * [ ]  Get the change log OK'd by other maintainers.
>       * [ ]  Publish the draft as a release.
>       * [ ]  Verify that the Helm repo was updated
>       * [ ]  Add a link to the tagged release in this issue.
> 
>     * [ ]  Close this issue.

We should probably add updating operator bundle too.

**Comment by fmuyassarov** [comment]
> > ```
> > * [ ]  Close this issue.
> > ```
> 
> We should probably add updating operator bundle too.

Here is the fix: https://github.com/containers/nri-plugins/pull/292

**Comment by klihub** [comment]
Published release: https://github.com/containers/nri-plugins/releases/tag/v0.4.1

**Comment by klihub** [comment]
> > > ```
> > > * [ ]  Close this issue.
> > > ```
> > 
> > 
> > We should probably add updating operator bundle too.
> 
> Here is the fix: #292

Thanks! LGTM'ed it. Could you take care of generating and submitting the operator bundle for this release ?

**Comment by fmuyassarov** [comment]
> > > > ```
> > > > * [ ]  Close this issue.
> > > > ```
> > > 
> > > 
> > > We should probably add updating operator bundle too.
> > 
> > 
> > Here is the fix: #292
> 
> Thanks! LGTM'ed it. Could you take care of generating and submitting the operator bundle for this release ?

Yes, it is on me.

**Comment by fmuyassarov** [comment]
> Thanks! LGTM'ed it. Could you take care of generating and submitting the operator bundle for this release ?

Done in https://github.com/k8s-operatorhub/community-operators/pull/4035.
